examples/nexrad/nexrad_copy.py

```diff
@@ -0,0 +1,155 @@
+import logging
+from datetime import datetime, timedelta
+import os
+import sys
+import argparse
+
+from carto.auth import APIKeyAuthClient
+from carto.sql import SQLClient
+from carto.sql import CopySQLClient
```

Comment:
Maybe we can import `CopySQLClient` together with `SQLClient` in the previous line 👼